WalletWasabi/Crypto/Attribute.cs

```diff
@@ -0,0 +1,21 @@
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using WalletWasabi.Crypto.Groups;
+
+namespace WalletWasabi.Crypto
```

Comment:
1. Move `WalletWasabi.Crypto.ZeroKnowledge.Credential` to to the same namespace?
2. Move both to `WalletWasabi.WabiSabi` namespace?

Reply:
I've removed this monstruosity